### DISCUSSION TITLE
domain ".rural" won't save

### DISCUSSION THREAD

**Discussion by ricricucit** [vercel/community#7356] [Help] [answered]
### Summary

As the screenshot demonstrate (after waiting 24h), my (special?) domain won't set up.

<img width="1830" alt="Screenshot 2024-07-03 at 13 17 49" src="https://github.com/vercel/community/assets/1016775/7b1c6a29-a1bf-4538-8cad-77056ed45d25">

anyone can hint me on waht could be wrong? thanks

### Example

try to save a ".rural" domain from namecheap

### Steps to Reproduce

1. go to namecheap
2. get a .rural domain
3. try to set it as domain in vercel

**Comment by pawlean** [comment, marked as answer]
Hi @ricricucit,

`.rural` domains are handshake domains that won't work properly with Vercel.  

![CleanShot 2024-07-03 at 13 39 37](https://github.com/vercel/community/assets/17087373/38122a49-fdd2-485f-9500-638f237aff4c)

Please choose another domain that's official supported.

[a comment by an automated account is not shown]
